README.md

```diff
@@ -19,10 +19,30 @@ Now, use the tool with `kubectl`
 kubectl rook-ceph ceph status
 ```
 
+### Examples
+
```

Comment:
What about the following changes for examples?
- Move the "Examples" after the "Usage" section (I know, another reorder :)
- Each example gets its own 3rd-level header and full output included with the command. For example:

## Examples
### Run a Ceph Command
Any `ceph` command can be run with the plugin. This example gets the ceph status:

```
$ kubectl rook-ceph ceph status

<paste full output from a sample cluster>
```

### Restart the Operator
```
$ kubectll rook-ceph operator restart

<paste output>
```
...

Reply:
updated

Reply:
Did you push the changes? I don't see it

Reply:
can see it now?